src/components/header/index.css

```diff
@@ -249,8 +249,8 @@
 }
 
 .gist-optional-input-container {
-  float: left;
-  width: 50%;
+  /* float: left; */
```

Comment:
remove if not necessary

Reply:
Actually I wanted to ask if you want to keep the filename and revision inputs in a row or like this:
![Screenshot from 2019-03-12 14-35-48](https://user-images.githubusercontent.com/30416891/54187722-35ed2400-44d4-11e9-97d7-e2b4ce28470b.png)